### PR TITLE
Update OH automation

### PR DESCRIPTION
**Pull request by mmuzikar**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/common/CommonSteps.java [hunk 1]
```diff
@@ -77,49 +83,48 @@ public static void undeploySyndesis() {
...
+                removableResources = removableResourcesList.size();
+            } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
+                e.printStackTrace();
+            }
+            log.info(String.format("Testing if project is clean, CRD instances: %d , RemovableResources: %d", crdInstances, removableResources));
+            return crdInstances == 0 &
+                removableResources <= 3;
+            // +1 because configMap can be there on OCP 4.7, see https://github.com/xtf-cz/xtf/issues/406 ,
+            // +2 resources from jaeger operator if there was a jaeger in this namespace before and there is another namespace with jaeger operator
+        }, TimeUnit.MILLISECONDS, WaitingConfig.timeoutCleanup(), "Cleaning project.");
```
Comment (mmuzikar) [inline utilities/src/main/java/io/syndesis/qe/common/CommonSteps.java]: @mkralik3 I think this is no longer needed right? I might've included it in the backport by accident

Comment (mkralik3) [reply utilities/src/main/java/io/syndesis/qe/common/CommonSteps.java]: @mmuzikar  correct, after XTF update, this isn't longer needed.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Review by mkralik3 — APPROVED** [review]

**Pull request merged**